zb-db/src/main/java/io/zeebe/db/ColumnFamily.java

```diff
@@ -159,7 +159,9 @@ void whileTrue(
   boolean exists(KeyType key);
 
   /**
-   * Checks if a key with prefix keyPrefix exists in the column family.
+   * <b>Note:</b> This operation is slow.
+   *
+   * <p>Checks if a key with prefix keyPrefix exists in the column family.
```

Comment:
I think we could also just delete it right? Or do we need it elsewhere?

Reply:
It is not used anywhere else. We can delete it.